### PR TITLE
Updated french translation

### PR DESCRIPTION
**Pull request by Tonus1**
I reviewed a few strings and added a few missing.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 "Project-Id-Version: newsboat 2.6\n"
 "Report-Msgid-Bugs-To: https://github.com/newsboat/newsboat/issues\n"
 "POT-Creation-Date: 2021-12-08 18:49+0300\n"
-"PO-Revision-Date: 2020-12-10 21:26+0100\n"
-"Last-Translator: Tanguy Kerdoncuff <t.kerdonc@gmail.com>\n"
-"Language-Team: Nicolas Martyanoff <khaelin@gmail.com, Tanguy Kerdoncuff <t."
-"kerdonc@gmail.com>\n"
+"PO-Revision-Date: 2022-03-03 14:28+0000\n"
+"Last-Translator: antoine <tonus1@gmail.com>\n"
+"Language-Team: French <kde-i18n-doc@kde.org>\n"
```
Comment (Minoru) [inline po/fr.po:13]: We don't have language teams in Newsboat, so we enumerate all the translators in this field. Please roll back this change, and add yourself.

File: po/fr.po [hunk 2]
```diff
@@ -1047,9 +1042,10 @@ msgid "No item selected!"
 msgstr "Aucun élément sélectionné !"
 
 #: src/itemlistformaction.cpp:108
-#, fuzzy
 msgid "Do you really want to delete all articles (y:Yes n:No)? "
-msgstr "Souhaitez-vous réellement écraser `%s' (y : Oui n : No)? "
+msgstr ""
+"Souhaitez-vous réellement supprimer tous les articles "
+"`%s' (y : Oui n : No)? "
```
Comment (Minoru) [inline po/fr.po:1048]: The original string doesn't contain `%s`; are you sure the translation is correct?

File: po/fr.po [hunk 3]
```diff
@@ -1246,7 +1242,7 @@ msgstr "Ajout de %s à la file d'attente."
 #: src/itemviewformaction.cpp:224
 #, c-format
 msgid "%s is already queued."
-msgstr ""
+msgstr "%s est déjà dans la file"
```
Comment (Minoru) [inline po/fr.po:1245]:
```suggestion
msgstr "%s est déjà dans la file."
```

File: po/fr.po [hunk 4]
```diff
@@ -1656,9 +1653,9 @@ msgid "`%s' is not a valid key command"
 msgstr "`%s' n'est pas un raccourci correct"
 
 #: src/keymap.cpp:775
-#, fuzzy, c-format
+#, c-format
 msgid "failed to parse operation sequence for %s"
-msgstr "Echec d'ouverture du fichier (%s)"
+msgstr "Echec d'analyse de la séquence d'opérations sur (%s)"
```
Comment (Minoru) [inline po/fr.po:1658]:
```suggestion
msgstr "Echec d'analyse de la séquence d'opérations sur %s"
```

File: po/fr.po [hunk 5]
```diff
@@ -1676,7 +1673,7 @@ msgid "regular expression '%s' is invalid: %s"
 msgstr "l'expression régulière '%s' est invalide : %s"
 
 #: src/opml.cpp:165
-#, fuzzy, c-format
+#, c-format
 msgid "Error: failed to parse OPML file \"%s\""
 msgstr "Erreur : échec d'analyse du fichier OPML %s"
```
Comment (Minoru) [inline po/fr.po:1678]:
```suggestion
msgstr "Erreur : échec d'analyse du fichier OPML \"%s\""
```

File: po/fr.po [hunk 6]
```diff
@@ -1738,7 +1735,7 @@ msgid "start download on startup"
 msgstr "commencer le téléchargement au démarrage"
 
 #: src/pbview.cpp:64
-#, fuzzy, c-format
+#, c-format
```
Comment (Minoru) [inline po/fr.po:1738]: The translation ends with `%s`, whereas the original message doesn't; are you sure the translation is correct?

File: po/fr.po [hunk 7]
```diff
@@ -1799,7 +1796,7 @@ msgid "Mark as Finished"
 msgstr "Marquer comme terminé"
 
 #: src/queueloader.cpp:127
-#, fuzzy, c-format
+#, c-format
```
Comment (Minoru) [inline po/fr.po:1799]: I can't mark the offending line, but "podboat" in the translated message should start with a capital letter.

File: po/fr.po [hunk 8]
```diff
@@ -1867,28 +1864,27 @@ msgid "URLs"
 msgstr "Liens"
 
 #: src/utils.cpp:707
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to open file: %s"
 msgstr "Echec d'ouverture du fichier (%s)"
```
Comment (Minoru) [inline po/fr.po:1869]:
```suggestion
msgstr "Echec d'ouverture du fichier: %s"
```

File: po/fr.po [hunk 9]
```diff
@@ -1867,28 +1864,27 @@ msgid "URLs"
...
 
 #: src/utils.cpp:711
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to read line %u: %s"
 msgstr "Echec de lecture de ligne %u (%s)"
```
Comment (Minoru) [inline po/fr.po:1874]:
```suggestion
msgstr "Echec de lecture de ligne %u: %s"
```

File: po/fr.po [hunk 10]
```diff
@@ -1867,28 +1864,27 @@ msgid "URLs"
...
 
 #: src/view.cpp:164
-#, fuzzy
 msgid "Error: failed to execute startup commands"
 msgstr "Erreur : Echec d'éxécution des commandes de démarrage"
```
Comment (Minoru) [inline po/fr.po:1878]:
```suggestion
msgstr "Erreur : echec d'éxécution des commandes de démarrage"
```

File: po/fr.po [hunk 11]
```diff
@@ -2007,7 +2003,6 @@ msgid "Internal parse error"
 msgstr "Erreur d'analyse interne"
 
 #: rust/libnewsboat/src/fslock.rs:57
-#, fuzzy
 msgid "Failed to open lock file '%s': %s"
 msgstr "Echec d'ouverture du fichier de verrouillage : '%s' (%s)"
```
Comment (Minoru) [inline po/fr.po:2007]: Please check this translation; the punctuation at the end of the messages is different.

File: po/fr.po [hunk 12]
```diff
@@ -2017,7 +2012,6 @@ msgid "<filename containing invalid UTF-8 codepoint>"
 msgstr "<fichier contenant un code UTF-8 invalide>"
 
 #: rust/libnewsboat/src/fslock.rs:77
-#, fuzzy
 msgid "Failed to write PID to lock file '%s': %s"
 msgstr "Echec d'écriture du PID dans le fichier de verrouillage : '%s' (%s)"
```
Comment (Minoru) [inline po/fr.po:2016]: Please check this translation, as the punctuation at the end differs between original and translation.

File: po/fr.po [hunk 13]
```diff
@@ -224,13 +223,12 @@ msgid "`%s' is not a valid configuration element"
 msgstr "`%s' n'est pas un élément de configuration valide"
 
 #: src/configcontainer.cpp:160
-#, fuzzy, c-format
+#, c-format
 msgid "Newsboat: finished reload, %f unread feeds (%n unread articles total)"
 msgstr ""
 "newsboat : rechargement terminé, %f fils à lire (%n articles à lire au total)"
```
Comment (Minoru) [inline po/fr.po:229]:
```suggestion
"Newsboat : rechargement terminé, %f fils à lire (%n articles à lire au total)"
```

File: po/fr.po [hunk 14]
```diff
@@ -224,13 +223,12 @@ msgid "`%s' is not a valid configuration element"
...
 
 #: src/configcontainer.cpp:289
-#, fuzzy
 msgid ""
 "%N %V - Articles in feed '%T' (%u unread, %t total)%?F? matching filter "
 "'%F'&? - %U"
```
Comment (Minoru) [inline po/fr.po:234]: Please update the translation — note that the punctuation around `F` is different.

**Review by Minoru — CHANGES_REQUESTED** [review]
Thank you for the pull request, and sorry I didn't respond to #1979 in time!

This looks quite nice, but there are a few messages that should've been updated when you removed the 'fuzzy" tag. I commented on the ones that I spotted, but I don't speak French so I could have missed some. Please re-read them all and make sure that the translation *really* matches the English original, including the capitalization and punctuation.

**Comment by Tonus1** [comment]
I'm quite new to pull requests and corrected all that in my file... I will try a new pull-request.
Sorry for the mess !

**Pull request closed without merging**